conformance_automated = "Fail"
errors_diff = """
Lines 88, 89: Expected error (tag 'Customer3Subclass')
Line 52: Unexpected errors ['dataclasses_transform_func.py:52:8: error[missing-argument] No arguments provided for required parameters `id`, `name`', 'dataclasses_transform_func.py:52:18: error[too-many-positional-arguments] Too many positional arguments: expected 0, got 2']
Line 76: Unexpected errors ['dataclasses_transform_func.py:76:36: error[invalid-return-type] Function always implicitly returns `None`, which is not assignable to return type `T@create_model_frozen`']
"""
output = """
dataclasses_transform_func.py:52:8: error[missing-argument] No arguments provided for required parameters `id`, `name`
dataclasses_transform_func.py:52:18: error[too-many-positional-arguments] Too many positional arguments: expected 0, got 2
dataclasses_transform_func.py:56:1: error[invalid-assignment] Object of type `Literal[3]` is not assignable to attribute `name` of type `str`
dataclasses_transform_func.py:60:6: error[unsupported-operator] Operator `<` is not supported for types `Customer1` and `Customer1`
dataclasses_transform_func.py:64:36: error[unknown-argument] Argument `salary` does not match any known parameter
dataclasses_transform_func.py:70:8: error[missing-argument] No arguments provided for required parameters `id`, `name`
dataclasses_transform_func.py:70:18: error[too-many-positional-arguments] Too many positional arguments: expected 0, got 2
dataclasses_transform_func.py:76:36: error[invalid-return-type] Function always implicitly returns `None`, which is not assignable to return type `T@create_model_frozen`
dataclasses_transform_func.py:96:1: error[invalid-assignment] Property `id` defined in `Customer3` is read-only
"""
